allauth_2fa/adapter.py

```diff
@@ -11,7 +11,7 @@ def login(self, request, user):
 
         # Require two-factor authentication if it has been configured.
         if user.totpdevice_set.all():
-            request.session['user_id'] = user.id
+            request.session['allauth_2fa_user_id'] = user.id
```

Comment:
Are there any backwards compatibility concerns with currently open sessions that might have `user_id` set?